---
layout: article
title: Links to datasets
---

## From papers

#### (Rahu et al, 2025) [(doi)](https://doi.org/10.1093/genetics/iyaf210)
* [Summary statistics](https://doi.org/10.5281/zenodo.13821209)
* [SuSiE fine mapping results](https://doi.org/10.5281/zenodo.13821038)

#### MetaLCL (Freimann et al, 2025) [(doi)](https://doi.org/10.1038/s41467-025-63856-7)
* [Project repository](https://github.com/AlasooLab/MetaLCL)
* [Summary statistics](https://ftp.ebi.ac.uk/pub/databases/spot/eQTL/imported/MetaLCL/)

#### gpu-coloc (Jesse et al, 2025) [(doi)](https://doi.org/10.1101/2025.08.25.672103)
* [Pre-formatted eQTL Catalogue LBFs](https://tartuulikool-my.sharepoint.com/:f:/r/personal/a72094_ut_ee/Documents/eQTL_Catlogue_gpu-coloc?csf=1&web=1&e=U6OhP4)

#### EstBB-UKBB meta-analysis (Tambets et al, 2024) [(doi)]()
* [Project repostiory](https://github.com/ralf-tambets/EstBB-UKBB-metaanalysis)
* [Summary statistics](https://github.com/ralf-tambets/EstBB-UKBB-metaanalysis/blob/main/data/sumstats_paths.tsv)

#### (Kolberg et al, 2020) [(doi)](https://doi.org/10.7554/eLife.58705)
* [Co-expression trans-eQTL analysis data](https://doi.org/10.5281/zenodo.3759693)

#### eQTL Catalogue [(doi)](https://doi.org/10.1101/2020.01.29.924266)
* [Summary statistics and fine mapping credible sets](https://www.ebi.ac.uk/eqtl/Data_access/)
* [Reference transcriptome annotations](https://doi.org/10.5281/zenodo.3366280)
* [Metadata for molecular phenotypes](https://doi.org/10.5281/zenodo.3366011)
* [Raw microarray data](https://doi.org/10.5281/zenodo.3565554)

#### (Alasoo et al, 2019) [(doi)](https://doi.org/10.7554/eLife.41673.001)
* [RNA-seq read counts](https://doi.org/10.5281/zenodo.2571453)
* [QTL lead variants](https://doi.org/10.5281/zenodo.1245778)
* [Full QTL summary statistics](https://doi.org/10.5281/zenodo.1239670) [(part 2)](https://doi.org/10.5281/zenodo.1245513)
* [Colocalisation results](https://doi.org/10.5281/zenodo.1245544)
* [Coordinates of genetic variants](https://doi.org/10.5281/zenodo.1245602)
* [Differential gene expression results](https://doi.org/10.5281/zenodo.1245133)
* [Transcriptional events constructed by txrevise](https://doi.org/10.5281/zenodo.3232932)

#### (Alasoo et al, 2018) [(doi)](http://dx.doi.org/10.1038/s41588-018-0046-7)
* [RNA-seq and ATAC-seq read counts](https://zenodo.org/record/259661)
* [Differential expression results](https://zenodo.org/record/839011)
* [Flow cytometry data](https://zenodo.org/record/234214)
* [ATAC-seq Bioanalyzer profiles](https://zenodo.org/record/242383)

#### (Alasoo et al, 2015) [(doi)](http://www.nature.com/articles/srep12524)
* [RNA-seq read counts](https://zenodo.org/record/16739)
* [Flow cytometry data](https://doi.org/10.6084/m9.figshare.1119735)

## Other datasets
1. [Transcriptional events constructed by txrevise](https://doi.org/10.5281/zenodo.3232932)
1. [Yeast gene expression datasets with curated annotations](https://zenodo.org/record/883139)